paperweight-lib/src/main/kotlin/tasks/RemapSources.kt

```diff
@@ -158,12 +165,29 @@ abstract class RemapSources : JavaLauncherTask() {
 
             zip(srcOut, sourcesOutputZip)
             zip(testOut, testsOutputZip)
+
+            writeSpigotRecompiledFiles(srcOut)
         } finally {
             srcOut.deleteRecursively()
             testOut.deleteRecursively()
         }
     }
 
+    private fun writeSpigotRecompiledFiles(srcOut: Path) {
+        // Write list of java files spigot recompiles
+        val spigotRecompiled = Files.walk(srcOut).use { stream ->
+            stream.map {
```

Comment:
Might be silly, but if you do `stream.asSequence()` then you can use `mapNotNull` and `joinToString("\n")`, which looks a little nicer than a Java stream. But I also don't care and it doesn't matter.